src/contractTest/java/uk/gov/hmcts/reform/et/syaapi/consumer/ccd/SubmitEventForCitizenConsumerTest.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.reform.et.syaapi.consumer.ccd;
+
+import au.com.dius.pact.consumer.dsl.DslPart;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.et.common.model.ccd.Et1CaseData;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.et.syaapi.consumer.SpringBootContractBaseTest;
+import uk.gov.hmcts.reform.et.syaapi.utils.ResourceLoader;
+
+import java.util.Map;
+
+import static io.pactfoundation.consumer.dsl.LambdaDsl.newJsonBody;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class SubmitEventForCitizenConsumerTest extends SpringBootContractBaseTest {
+
+    @Pact(provider = "ccd_data_store_api_cases", consumer = "et_sya_api_service")
+        public RequestResponsePact submitEventForCitizen(PactDslWithProvider builder) {
+
+        Map<String, String> responseHeaders = Map.of("Content-Type", "application/json");
+
+        return  builder
+                .given("A Submit event for a Citizen is requested", addCaseTypeJurdisticaton())
+                .uponReceiving("A Submit event for a Citizen against CCD API")
+                .path(buildPath())
+                .query("ignore-warning=true")
+                .method("POST")
+                .headers(responseHeaders)
+                .matchHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .willRespondWith()
+                .matchHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .status(201)
+                .body(buildCaseDetailsDsl(CASE_ID))
+                .toPact();
+    }
+
+    @Test
+    @SneakyThrows
+    @PactTestFor(pactMethod = "submitEventForCitizen")
+    void verifysubmitEventForCitizen() {
+        Et1CaseData caseData = ResourceLoader.fromString(
+            "requests/caseData.json",
+            Et1CaseData.class
+        );
```

Comment:
Javascript pattern, Keep bracket on same line as code.
Please review all of this PR for instances of this :)